js/app/packages/app/component/next-soup/soup-view/views/tasks/list-property-value.css

```
@@ -2,14 +2,24 @@
   --task-col-status: 7rem;
   --task-col-priority: 7rem;
   --task-col-assignees: 7rem;
+  --task-col-created-by: 7rem;
   --task-col-timestamp: 5rem;
 }
 
+/* Medium containers: hide Created By column */
+@container uList (max-width: 1220px) {
```

Stylelint (17.11.0)

[error] Expected container name "uList" to be kebab-case (container-name-pattern)

(container-name-pattern)